### PR TITLE
Default Branch Checking Bugfix

### LINKED ISSUE #169: Failed to run automated scorecard-action@main ossf-tests/scorecard-action-non-main-branch
https://github.com/ossf-tests/scorecard-action-non-main-branch/actions/runs/2142139482

### PR DESCRIPTION
**Pull request by rohankh532**
PR to fix #169

### CODE REVIEW

**Review by naveensrinivasan — APPROVED** [review]
Looks good. @azeemshaikh38 👀

**Comment by naveensrinivasan** [comment]
Shouldn't this target golang-staging branch?

**Comment by rohankh532** [comment]
> Shouldn't this target golang-staging branch?

Oops, fixed.

**Comment by azeemshaikh38** [comment]
> > Shouldn't this target golang-staging branch?
> 
> Oops, fixed.

This should not be `golang-staging`, merging this to `main` is the right thing to do. Copying my reply to Naveen over slack here: 

> we want to merge these changes to main  and after that to golang-staging . Basically, the only diff between main  and golang-staging  should be that the Dockerfile uses Golang instead of Bash. Everything else should be equivalent

**Comment by rohankh532** [comment]
> > > Shouldn't this target golang-staging branch?
> > 
> > 
> > Oops, fixed.
> 
> This should not be `golang-staging`, merging this to `main` is the right thing to do. Copying my reply to Naveen over slack here:
> 
> > we want to merge these changes to main  and after that to golang-staging . Basically, the only diff between main  and golang-staging  should be that the Dockerfile uses Golang instead of Bash. Everything else should be equivalent

Got it, changed back to main.

File: Dockerfile [hunk 1]
```diff
@@ -21,23 +21,30 @@
...
+WORKDIR /src
+ENV CGO_ENABLED=0
+COPY go.* ./
+RUN go mod download
+COPY . ./
+
+FROM base AS build
+ARG TARGETOS
+ARG TARGETARCH
+RUN CGO_ENABLED=0 make build
```
Comment (azeemshaikh38) [inline Dockerfile:36]: Dockerfile shouldn't be updated?

**Review by azeemshaikh38 — CHANGES_REQUESTED** [review]
LGTM overall. Revert the changes on Dockerfile and ensure that the unit tests are passing.

File: Dockerfile (hunk 1 above)
Comment (rohankh532) [reply Dockerfile]: Reverted this

**Review by azeemshaikh38 — APPROVED** [review]
Thanks!

**Pull request merged**